src/pudl/metadata/classes.py

```diff
@@ -907,8 +909,8 @@ class DataSource(Base):
     keywords: List[str] = []
     path: HttpUrl = None
     contributors: List[Contributor] = []  # Or should this be compiled from Resources?
-    license_raw: License
-    license_pudl: License
+    license_raw: License = None
+    license_pudl: License = None
```

Comment:
When I filtered the resources included by `etl_group` I use `DataSource.get_resource_ids()` for each included `etl_group`. I created a list of ETL groups that are sort of "extra" and aren't an actual data source that is used in Datasette (`extra_etl_groups = ['entity_eia', 'glue', 'static_eia', 'static']`). For these extra data source names the `DataSource` instance has no other attributes besides `name`, so I had to make None the default for License. I could attach `license_raw = LICENSES["us-govt"], license_pudl = LICENSES["cc-by-4.0"]` to these data sources instead since they're not used after the call to `get_resource_ids()`